Resources/public/js/views/ez-contenttypeselectorview.js

```diff
@@ -194,7 +194,6 @@ YUI.add('ez-contenttypeselectorview', function (Y) {
 
             this.fire('createContent', {
                 contentType: type,
-                languageCode: 'eng-GB',
```

Comment:
that's a BC break, you can not do that, we have to keep `languageCode`. As I was writing in https://github.com/ezsystems/PlatformUIBundle/pull/581#discussion_r62484406, you should properly document the event and explain that the `languageCode` is deprecated (example of doc block for the event can be found in http://ezsystems.github.io/platformui-javascript-api/files/Resources_public_js_views_ez-serversideview.js.html#l72 for example).